### PR TITLE
Implement SessionExpireHandler

### PR DESCRIPTION
**Pull request by nicktogo**
- #3 
- change strings to constant

### CODE REVIEW

**Comment by nicktogo** [comment]
also fixed bug in #2

File: Server/src/main/java/octoteam/tahiti/server/pipeline/MessageForwardHandler.java [hunk 1]
```diff
@@ -17,12 +17,14 @@ public MessageForwardHandler(TahitiServer server) {
...
-        this.server.getAllConnected().flushAndWrite(resp.build());
+        if (message.getService().equals(Message.ServiceCode.CHAT_SEND_MESSAGE_REQUEST)) {
+            Message.Builder resp = Message
+                    .newBuilder()
+                    .setSeqId(message.getSeqId())
+                    .setDirection(Message.DirectionCode.EVENT)
+                    .setService(Message.ServiceCode.CHAT_BROADCAST_EVENT)
+                    .setChatMessageReq(message.getChatMessageReq());
+            this.server.getAllConnected().flushAndWrite(resp.build());
+        }
```
Comment (SummerWish) [inline Server/src/main/java/octoteam/tahiti/server/pipeline/MessageForwardHandler.java]: Not forwarding other messages to the next handler

File: Server/src/main/java/octoteam/tahiti/server/pipeline/SessionExpireHandler.java [hunk 2]
```diff
@@ -2,21 +2,36 @@
...
+                Message.Builder resp = Message.newBuilder()
+                        .setDirection(Message.DirectionCode.RESPONSE)
+                        .setService(Message.ServiceCode.SESSION_EXPIRED_EVENT);
+                ctx.channel().writeAndFlush(resp.build());
+                setSession(ctx, null);
+            } else {
+                Message.Builder resp = Message.newBuilder()
+                        .setDirection(Message.DirectionCode.RESPONSE)
+                        .setStatus(Message.StatusCode.LIMIT_EXCEEDED);
+                ctx.channel().writeAndFlush(resp.build());
```
Comment (SummerWish) [inline Server/src/main/java/octoteam/tahiti/server/pipeline/SessionExpireHandler.java]: SessionExpireHandler should not send LIMIT_EXCEEDED and LIMIT_EXCEEDED is already sent in [RateLimitHandler](https://github.com/SummerWish/Tahiti/blob/master/Server/src/main/java/octoteam/tahiti/server/pipeline/RateLimitHandler.java#L35)

**Comment by nicktogo** [comment]
All done.

**Comment by SummerWish** [comment]
Good job! Now you could write tests for SessionExpireHandler.

**Pull request merged**